pkgs/by-name/so/sold/package.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "sold";
+  version = "1.11.0";
```

As per https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming
The upstream has not made any releases or attached tags, so this should be
```suggestion
  version = "0-unstable-2024-02-22";
```

---

Thank you, I have resolved this